sensor/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,118 @@
+# tag_based_pnp_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/tag_based_pnp_calibrator.md).
+
+## Purpose
+
+The package `tag_based_pnp_calibrator` allows extrinsic calibration among the camera and lidar sensors used in autonomous driving and robotics.
+
+## Inner-workings / Algorithms
+
+The `tag_based_pnp_calibrator` utilizes the PnP (Perspective-n-Point) algorithm, a computer vision method that finds the best match between a set of 3D points (from the lidar) and their corresponding 2D projections (in the camera images), to accurately calculate the transformation between the camera and lidar. To run this package, you also need to execute the `apriltag_ros` and the `lidartag` packages to calculate the transformation.
+
+The `apriltag_ros` package detects AprilTag markers from an image and outputs the detection results. Conversely, the `lidartag` package detects LidarTag markers and outputs its detection results.
+
+The `tag_based_pnp_calibrator` utilizes the detections from both `apriltag_ros` package and `lidartag` package, employing a Kalman Filter to track these detections. If the detections converge, the calibrator applies the SQPnP algorithm provided by OpenCV to estimate the transformation between the image points from AprilTag and the object points from LidarTag.
```

Comment:
(optional)
Reference to the sqpnp paper

Reply:
Thanks!
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/002a39c2ca0808ab85521e054657559c817f4bae

Reply:
This is not a paper, and we are not following any citation standards really, but there is relevant information missing, like where and when it was published

https://link.springer.com/chapter/10.1007/978-3-030-58452-8_28#citeas

@vividf

Reply:
add them in https://github.com/tier4/CalibrationTools/pull/164/commits/b0b7c86085e42a8616f632c92b0915cf761cd779

Reply:
That empty () is a format error

Reply:
removed in https://github.com/tier4/CalibrationTools/pull/164/commits/3b37a80fa659ae4a00b3c60d2072a000a22f4822